app/views/users/_users.html.erb

```diff
@@ -1,4 +1,14 @@
-  <tr>
-    <td><%= id + 1 %></td>
-    <td><%= user.email %></td>
-  </tr>
+<tr>
+  <td><%= id + 1 %></td>
```

Comment:
I'm not sure what I should change. Did you mean change color/style table or set image as background ?